src/styles/_catalog.scss

```
@@ -14,6 +14,14 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition: color, transform 300ms;
```

color doesn't have transition now
```suggestion
    transition: color 300ms, transform 300ms;
```